test/basic.js

```diff
@@ -265,16 +266,52 @@ function createInferElectronTest (opts) {
   }
 }
 
+function copyFixtureToTempDir (fixtureSubdir, cb) {
+  let tmpdir = path.join(os.tmpdir(), fixtureSubdir)
+  let fixtureDir = path.join(__dirname, 'fixtures', fixtureSubdir)
+  waterfall([
+    cb => fs.emptyDir(tmpdir, cb),
+    (cb1, cb2) => fs.copy(fixtureDir, tmpdir, cb2 || cb1), // inconsistent cb arguments from fs.emptyDir
```

Comment:
This is weird b/c fs.emptyDir doesn't return a consistent # of arguments, depending on whether the directory already exists or not. I've made an issue on the `fs-extra` repo but this is the workaround for now.